service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsController.java

```diff
@@ -156,9 +161,41 @@ public CallbackResponse validateHearingDatesMidEvent(@RequestBody CallbackReques
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
         CaseData caseData = getCaseData(caseDetails);
 
-        List<String> errors = validateGroupService.validateGroup(caseData, HearingDatesGroup.class);
+        List<String> errors;
+
+        if (featureToggleService.isAddHearingsInPastEnabled() && (caseData.getHearingOption() == NEW_HEARING
+            || isEmpty(caseData.getHearingOption()))) {
```

Comment:
minor: what about refactoring here just for the sake of readability. 

`featureToggleService.isAddHearingsInPastEnabled() && isAddingNewHearing(caseData)`

```
    private boolean isAddingNewHearing(CaseData caseData) {
        return isEmpty(caseData.getHearingOption()) || NEW_HEARING.equals(caseData.getHearingOption());
    }
```